src/parser_urdf_TEST.cc

```diff
@@ -994,6 +1008,553 @@ TEST(URDFParser, ParseWhitespace)
   EXPECT_EQ("1000", std::string(mu2Elem->GetText()));
 }
 
+/////////////////////////////////////////////////
+TEST(URDFParser, DefaultWarnsWhenIgnoringLinksWithNoInertia)
+{
+  // Capture sdferr output
+  std::stringstream buffer;
+  auto old = std::cerr.rdbuf(buffer.rdbuf());
+
+  #ifdef _WIN32
+    sdf::Console::Instance()->SetQuiet(false);
```

Comment:
Use `ScopeExit` to reset this back to true.

Reply:
https://github.com/gazebosim/sdformat/pull/1238/commits/d4de27510d7e37294d2809319cbabc36cc74a9f1